tutorial_4_django_views_templates/slides.md

```diff
@@ -0,0 +1,130 @@
+Tutorial: Create Views and Route your URLS(urls setting)
+===========================================
+[Demonstration time: 7 mins 00 s (0.817 ~ 82%) | Total time: 8 mins 34 s]
+
+Slide 1 [00:08 | 00:08]
+------------
+Title Slide
+**Creating Views and Routing URLs**
+
+Slide 2 [00:12 | 00:20]
+--------------
+
+**Learning Objectives**
+
+In this tutorial, we will learn to;
+  - Create a django view
+  - Create a url routing scheme
+
+Slide 3 [00:11 | 00:31]
+---------------
+
+**System Requirements**
+  - Ubuntu 16.10
+  - Python 3.5 or higher version
+  - python3.4-venv
+  
+Slide 4 [00:11 | 00:42]
+---------------
+
+**Pre-requisites**
+
+In order to follow this tutorial, you need to know;
+  - how to create models in django
+  - If not, see the relevant django tutorial on http://spoken-tutorial.org
```

Comment:
Yes, links will redirect to the relevant tutorials. Currently links are acting as placeholders